sos/plugins/sclnpm.py

```diff
@@ -0,0 +1,14 @@
+from sos.plugins import Plugin, SCLPlugin
+
+class SCLNpm(Plugin, SCLPlugin):
```

Comment:
The Travis CI is failing because you need to insert one more blank line between the import and SCLNpm class.

Per [Contribution Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines) your commit also needs to be accompanied by a Signed-off-by.